libnmstate/nm/applier.py

```diff
@@ -123,6 +123,7 @@ def set_ifaces_admin_state(ifaces_desired_state, con_profiles=()):
     new_vlan_ifaces_to_activate = set()
     new_ovs_interface_to_activate = set()
     new_ovs_port_to_activate = set()
+    master_ifaces_to_add = set()
```

Comment:
The missing info here is: `new`, `not enslaved/slave`.
`new_master_non_slave_ifaces` OR
`new_master_not_enslaved_ifaces`?

Reply:
Done.